install.ps1

```diff
@@ -1,429 +1,931 @@
+    $unselectedPackagesBox.width         = 246
+    $unselectedPackagesBox.height        = 300
+    $unselectedPackagesBox.location      = New-Object System.Drawing.Point(6,65)
+
+    $packageTypeCombo                  = New-Object system.Windows.Forms.ComboBox
+    $packageTypeCombo.width            = 246
+    $packageTypeCombo.height           = 20
+    $packageTypeCombo.location         = New-Object System.Drawing.Point(6,40)
+    $packageTypeCombo.Font             = New-Object System.Drawing.Font('Microsoft Sans Serif',10)
+    $packageTypeCombo.Items.Add('All') | Out-Null
+    $packageTypeCombo.SelectedIndex    = 0
+
+    $removePackageButton               = New-Object system.Windows.Forms.Button
+    $removePackageButton.text          = "<"
+    $removePackageButton.width         = 24
+    $removePackageButton.height        = 26
+    $removePackageButton.location      = New-Object System.Drawing.Point(258,170)
+    $removePackageButton.Font          = New-Object System.Drawing.Font('Microsoft Sans Serif',10)
+    $removePackageButton.Add_Click({Remove-SelectedPackages})
+
+    $removeAllPackageButton            = New-Object system.Windows.Forms.Button
+    $removeAllPackageButton.text       = "<<"
+    $removeAllPackageButton.width      = 24
+    $removeAllPackageButton.height     = 26
+    $removeAllPackageButton.location   = New-Object System.Drawing.Point(258,140)
+    $removeAllPackageButton.Font       = New-Object System.Drawing.Font('Microsoft Sans Serif',7,[System.Drawing.FontStyle]::Bold)
+    $removeAllPackageButton.Add_Click({Remove-AllPackages})
+
+    $addPackageButton                 = New-Object system.Windows.Forms.Button
+    $addPackageButton.text            = ">"
+    $addPackageButton.width           = 24
+    $addPackageButton.height          = 26
+    $addPackageButton.location        = New-Object System.Drawing.Point(258,206)
+    $addPackageButton.Font            = New-Object System.Drawing.Font('Microsoft Sans Serif',10)
+    $addPackageButton.Add_Click({Add-SelectedPackages})
+
+    $addAllPackageButton              = New-Object system.Windows.Forms.Button
+    $addAllPackageButton.text         = ">>"
+    $addAllPackageButton.width        = 24
+    $addAllPackageButton.height       = 26
+    $addAllPackageButton.location     = New-Object System.Drawing.Point(258,236)
+    $addAllPackageButton.Font         = New-Object System.Drawing.Font('Microsoft Sans Serif',7,[System.Drawing.FontStyle]::Bold)
+    $addAllPackageButton.Add_Click({Add-AllPackages})
+
+    $dontInstallLabel                = New-Object system.Windows.Forms.Label
+    $dontInstallLabel.text           = "Available to Install"
+    $dontInstallLabel.AutoSize       = $true
+    $dontInstallLabel.width          = 25
+    $dontInstallLabel.height         = 10
+    $dontInstallLabel.location       = New-Object System.Drawing.Point(7,20)
+    $dontInstallLabel.Font           = New-Object System.Drawing.Font('Microsoft Sans Serif',10)
```

Comment:
Maybe comment this out for now as we only have "All" and it can be confusing what to select here.

Reply:
I removed the "All" dropdown for now, we can add categories in the future